components/filter/filter-dimension-set.js

```diff
@@ -120,6 +126,7 @@ class FilterDimensionSet extends LitElement {
 				disabled: value.disabled,
 				key: value.key,
 				selected: value.selected,
+				selectedOnRender: false,
```

Comment:
Is this needed? Seems like this is only set and controlled by the filter itself, and the value isn't used before it's initialized to the proper thing